src/ploneintranet/workspace/browser/add_content.py

```diff
@@ -34,13 +38,25 @@ def create(self):
             title=self.title,
             safe_id=True,
         )
+        if image:
+            namedblobimage = NamedBlobImage(
+                data=image.read(),
+                filename=safe_unicode(image.filename))
+            new.image = namedblobimage
+
         if new:
             new.description = safe_unicode(self.description)
             return new.absolute_url()
 
     def redirect(self, url):
         """Has its own method to allow overriding"""
-        url = '{}/edit'.format(url)
+        url = '{}/view'.format(url)
+        # portal_type = new.portal_type.lower()
+        # if portal_type in ['image', 'file']:
+        #     url = '{}/{}_view'.format(new_url, portal_type)
+        # else:
+        #     url = '{}'.format(new_url)
+        # import pdb; pdb.set_trace()
```

Comment:
Please don't commit commented code. VCS allows you to get it back later if you really need it, but eventually someone will look at it and either not know why it was commented and it will remain forever, with no understanding as to why it's there.